### PR TITLE
invidious: unstable-2024-01-29 -> unstable-2024-02-18

### PR DESCRIPTION
**Pull request by GaetanLepage**
## Description of changes

cc @infinisil @sbruder 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review pr 289929` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>invidious</li>
  </ul>
</details>

**Review by 999eagle — APPROVED** [review]
Deployed to my instance, LGTM!

File: pkgs/servers/invidious/versions.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
     "sha256": "sha256-EU6T9yQCdOLx98Io8o01rEsgxDFF/Xoy42LgPopD2/A="
   },
   "invidious": {
-    "rev": "c005ada48723808e507d0a4d5a3363a1c14a4f07",
-    "sha256": "sha256-KbnBdAAjScwKV4uUzyBXAQx2C7MqCdCM3gSvgNIzKAU=",
-    "version": "unstable-2024-01-29"
+    "rev": "e8a36985aff1a5b33ddf9abea85dd2c23422c2f7",
+    "sha256": "sha256-3nU6z1rd1oiNmIz3Ok02xBsT4oNSGX/n+3/WbRVCbhI=",
+    "version": "unstable-2024-02-18"
```
[the remark this answers is not in the document]
Comment (999eagle) [reply pkgs/servers/invidious/versions.json:9]: Yes this is generated using `pkgs/servers/invidious/update.sh`. We probably should update the update script for this.

[the remark this answers is not in the document]
Comment (GaetanLepage) [reply pkgs/servers/invidious/versions.json:9]: Done in https://github.com/NixOS/nixpkgs/pull/289937

[the remark this answers is not in the document]
Comment (999eagle) [reply pkgs/servers/invidious/versions.json:9]: I think what @kirillrdy meant was this: https://github.com/NixOS/nixpkgs/pull/234201
The new version should thus be `0.20.1-unstable-2024-02-18` to include the previous version. I'm currently preparing a PR to change the update script for this.

[the remark this answers is not in the document]
Comment (999eagle) [reply pkgs/servers/invidious/versions.json:9]: I've opened https://github.com/NixOS/nixpkgs/pull/289943 now to change the update script so the new versioning is used, update invidious and drop lsquic.

**Comment by GaetanLepage** [comment]
Superseded by #289943

File: pkgs/servers/invidious/versions.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (kirillrdy) [reply pkgs/servers/invidious/versions.json:9]: @999eagle thank you ! thats what I meant, but I also like https://github.com/NixOS/nixpkgs/pull/289937, maybe we should revisit it once https://github.com/NixOS/nixpkgs/pull/289943 is merged

**Pull request closed without merging**